cmd-display-menu.c

```diff
@@ -393,6 +396,23 @@ cmd_display_popup_exec(struct cmd *self, struct cmdq_item *item)
 	if (!cmd_display_menu_get_position(tc, item, args, &px, &py, w, h))
 		return (CMD_RETURN_NORMAL);
 
+	value = args_get(args, 'b');
+	if (args_has(args, 'B'))
+		lines = POPUP_LINES_NONE;
+	else if (value == NULL)
+		lines = options_get_number(o, "popup-border-lines");
+	else {
+		cause = NULL;
```

Comment:
💥 Not seeting `cause` to `NULL` here crashes the server, I'm not sure why.
@nicm Do you see what might be the issue here?